payload/.cargo/config

```diff
@@ -0,0 +1,7 @@
+[build]
+target = "x86_64-unknown-linux-musl"
+rustflags = [
+    "-C", "relocation-model=pic",
+    "-C", "opt-level=3",
```

Comment:
I don't think we need `opt-level=3` here by default.

Reply:
Addressing this in my next push